charm_versions.md

```diff
@@ -0,0 +1,138 @@
+# Versions
+
+## Juju Version
+
+The Juju version uses [semantic versioning](https://semver.org/), so is a string in the format
+major.minor.patch, where each section is a positive integer (which may be higher than 9).
+
+### Agent
+
+You can see the version of Juju used in a model in `juju status`, in the top section under
+"Version" in the tabular view, or `model.version` in JSON or YAML output. You can also use the
+`juju model-config -m controller agent-version` command.
+
+To upgrade the patch version of Juju (e.g. 3.3.0 to 3.3.1), you run
+[`juju upgrade-controller`](https://juju.is/docs/juju/juju-upgrade-controller), specifying the
+patch version. To upgrade minor or major versions, you need to create a new controller and
+[migrate](https://juju.is/docs/juju/juju-migrate) the models to it.
+
```

Comment:
This can be combined with https://juju.is/docs/juju/manage-logs to form a more comprehensive set of how-to guides "Manage agents".  PS Note we already have guides on how to upgrade / migrate a model and how to upgrade a controller: https://juju.is/docs/juju/manage-controllers#heading--upgrade-a-controller , https://juju.is/docs/juju/manage-models#heading--upgrade-a-model